sdk/attestation/Azure.Security.Attestation/src/Generated/Internal/MultipartFormDataRequestContent.cs

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
```

Comment:
Why is this type emitted in a library that never uses it?

Reply:
this is currently by-design unfortunately.
The generator works in this way: it cannot figure out whether a class is used anywhere until everything is generated, therefore the generator is relying a component in the generator called "post processor" to trim off those unused types.
Unfortunately this was not enabled for HLC libraries (the old data plane packages), therefore we will see this unused internal type getting into the library.
This is not the first time we have it - HLC has a different code structure, therefore it will not need any helpers in this directory: https://github.com/azure-sdk/azure-sdk-for-net/tree/e02c9947820d614362891f196ce1a2357dc4511c/sdk/attestation/Azure.Security.Attestation/src/Generated/Internal
We consider since it is internal, we just let it be.
We could do something to fix that, either from the generator side or in the generated library side that we could use the attribute to suppress them.